src/liberty/devCommands.ts

```diff
@@ -193,11 +192,11 @@ export async function addProject(uri: vscode.Uri): Promise<void> {
         // scan the folder and get a list of folders with pom.xml and build.gradle
         const uris: string[] = await projectProvider.getListOfMavenAndGradleFolders(uri.fsPath);
         console.log(JSON.stringify(uris));
-        if ( uris.length > 0) {
+        if (uris.length > 0) {
```

Comment:
Please do not do any formatting changes to code that is not added or modified by you for this PR.